superset/assets/src/dashboard/util/dashboardLayoutConverter.js

```diff
@@ -88,13 +88,13 @@ function getChartHolder(item) {
     Math.round(size_y / GRID_RATIO * 100 / ROW_HEIGHT),
   );
   if (code !== undefined) {
-    let markdownContent = '';
-    if (slice_name) {
-      markdownContent = `##### **${slice_name}**\n`;
-    }
+    let markdownContent = ' '; // white-space markdown
     if (code) {
-      markdownContent += code;
+      markdownContent = code;
+    } else if (slice_name.trim()) {
+      markdownContent = `##### **${slice_name}**\n`;
```

Comment:
Couple thoughts
- You shouldn't need the extra asterisks `**` surrounding `slice_name` because `###` is already bold
- I don't think you need the `\n` either since we are _only_ showing the title

Reply:
fixed. removed ** and new line.